docs/security-considerations.rst

```diff
@@ -31,6 +32,10 @@ Solidity compiler.
 As always, with open source documentation, please help us extend this section
 (especially, some examples would not hurt)!
 
+NOTE: In addition to the list below, you can find more security recommendations and best practices
+`here <https://github.com/guylando/KnowledgeLists/blob/master/EthereumSmartContracts.md>`_ and
+`here <https://consensys.github.io/smart-contract-best-practices/>`_.
```

Comment:
Could you please use a different word to highlight the second link. Alternatively you could use descriptive text for the links like so:

```
NOTE: In addition to the list below, you can find more security recommendations and best practices
`in Guy Lando's knowledge list <https://github.com/guylando/KnowledgeLists/blob/master/EthereumSmartContracts.md>`_ and
`the Consensys GitHub repo <https://consensys.github.io/smart-contract-best-practices/>`_.
```

Reply:
Just read up a little more about rst formatting. The words leading up to the angular brace forms the "hyperlink reference."

What our docs CI is saying is that you have two identically named hyperlink references. Although this may not be a problem in its current form, the `rst` docs say that

```
A hyperlink reference may directly embed a target URI or (since Docutils 0.11) a hyperlink reference within angle brackets ("<...>") as follows:

See the `Python home page <http://www.python.org>`_ for info.

This `link <Python home page_>`_ is an alias to the link above.
```

So, in this case, at a later point we could simply refer to one of the guides as `here.` If we do that, the `rst` builder will not know which of the two `here`s has to be linked. Hope this helps.